packages/api/src/command/medical/document/document-query.ts

```diff
@@ -66,12 +64,6 @@ export async function queryDocumentsAcrossHIEs({
     return createQueryResponse("processing", patient);
   }
 
-  const externalData = patient.data.externalData?.COMMONWELL;
-  if (!externalData) return createQueryResponse("failed");
-
-  const cwData = externalData as PatientDataCommonwell;
-  if (!cwData.patientId) return createQueryResponse("failed");
```

Comment:
#1192 shouldn't fail if no CW data, moved to within CW-specific data on `getDocumentsFromCW()`

Reply:
if there is no external data, I believe it means patient registration failed. If you follow breadcrumbs of creating a patient in cw leads here https://github.com/metriport/metriport/blob/e6fbc690838c76cb27c15e2c88acb8afe77f7159/packages/api/src/external/commonwell/patient-external-data.ts#L21

We didn't discuss this case (only the various linking status cases). Perhaps just something to be noted for my retry linking PR?

Reply:
It might mean the cx registration hasn't finished yet. I'd just make sure we're being notified if it fails - I think we are.